server_transcriptor/model_loader.py

```diff
@@ -1,18 +1,34 @@
 from sonosco.models import DeepSpeech2
-from sonosco.decoders import GreedyDecoder
+from sonosco.decoders import GreedyDecoder, BeamCTCDecoder
 from sonosco.datasets.processor import AudioDataProcessor
 
+
 # TODO: lazy loading
 def load_models(config):
     models = {}
+
     for model_config in config:
-        if model_config.get('script'):
-            pass  # TODO add possibility to add custom load script, not necessaryly in this way
+        if model_config.get('external'):
+            pass
         else:
-            model_dict = {}
-            models[model_config['id']] = model_dict
-            model = DeepSpeech2.load_model(model_config['path'])
-            model_dict['model'] = model
-            model_dict['decoder'] = GreedyDecoder(model.labels, blank_index=model.labels.index('_'))
-            model_dict['processor'] = AudioDataProcessor(**model.audio_conf, normalize=True)
+            model_dict = dict()
+            models[model_config["id"]] = model_dict
+            model = DeepSpeech2.load_model(model_config["path"])
+            model.eval()
+            model_dict["model"] = model
+
+            if model_config["decoder"] == "greedy":
+                model_dict["decoder"] = GreedyDecoder(model.labels, blank_index=model.labels.index('_'))
+            elif model_config["decoder"] == "beam":
+                if model_config.get("language"):
```

Comment:
You can get rid of this if and just to lm_path=model_config.get("language") the effect will be the same